litmus-portal/frontend/src/components/Sections/ChaosWorkflows/BrowseWorkflow/index.tsx

```diff
@@ -63,13 +63,16 @@ const BrowseWorkflow = () => {
   const { subscribeToMore, data, loading, error } = useQuery<
     Workflow,
     WorkflowDataVars
-  >(WORKFLOW_DETAILS, { variables: { projectID: '00000' } });
+  >(WORKFLOW_DETAILS, {
+    variables: { projectID: '00002' },
+    fetchPolicy: 'cache-and-network',
+  });
 
   // Using subscription to get realtime data
   useEffect(() => {
     subscribeToMore<WorkflowSubscription>({
       document: WORKFLOW_EVENTS,
-      variables: { projectID: '00000' },
+      variables: { projectID: '00002' },
```

Comment:
Go for 00000 if not being done dynamically

Reply:
This is just a variable for the query, its not hard-coded with the schema.